power-the-polls-form/package.json

```diff
@@ -19,9 +19,10 @@
    "scripts": {
       "dev": "stencil build --dev --watch --serve",
       "dev:site": "npm run clean && stencil build --watch",
-      "build": "npm run lint && stencil build --prod --docs --no-cache && terser ./dist/full/index.js -o ./dist/full/index.js",
+      "build": "npm run lint && stencil build --prod --docs --no-cache && terser ./dist/full/index.js -o ./dist/full/index.js && npm run get-data",
       "build:clean": "npm run clean-all && npm run build",
-      "package": "copyfiles -E -u 1 \"./dist/**/*\" \"../dist/power-the-polls-form/\" && copyfiles -E package.json Readme.md LICENSE ../dist/power-the-polls-form/ && replace \"dist/\" \"\" ../dist/power-the-polls-form/package.json && cross-env-shell replace \"CURRENT_VERSION\" \"$npm_package_version\" ../dist/power-the-polls-form/  -r --include=\"*.js\"",
+      "get-data": "cp \"src/data/full-jurisdictions.json\" \"./dist/collection/data/\" && cp \"src/data/full-jurisdictions.json\" \"./dist/types/data/\"",
```

Comment:
I had to manually do this as stencil does not have a way to do this using the stencil config. I used this guide as a breadcrumb trail. 
https://dockyard.com/blog/2019/05/23/stenciljs-lazy-loading-part-2-npm-build-assets